src/lib/passport-jwt.js

```diff
@@ -10,7 +10,7 @@ const options = {
 
 module.exports = (passport) => {
 	const verifyCallBack = async (jwtPayload, done) => {
-		const { email } = jwtPayload.user
+		const { email } = jwtPayload
```

Comment:
@beblicarl its the way the token is being generated in auth/token endpoint that is wrong. The jwt token should be signed in a similar way as the `/signin` and `create-user` were done. 

otherwise now the tokens from that one will also not work

Reply:
oh okay, i get you

Reply:
i have resolved this and pushed the changes